third_party/maplibre-native-qt/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+
+ARCHNAME="x86_64"
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  ARCHNAME="Darwin"
+fi
+
+if [ ! -d maplibre_repo ]; then
+  git clone git@github.com:maplibre/maplibre-native-qt.git $DIR/maplibre_repo
+fi
+cd maplibre_repo
+git fetch --all
+# git checkout (on main right now)
+git submodule update --depth=1 --recursive --init
```

Comment:
this should be pinned to a specific commit on their repo. in this case, likely the commit where they merged your fix